src/org/loklak/harvester/RedirectUnshortener.java

```diff
@@ -57,7 +57,7 @@
         "flip.it",
         "lnkd.in"
     };
```

Comment:
From the code I can see that it won't be, I am more over asking something to peek curiosity like, wouldn't that be a nicer thing.